functions-and-scope.js

```diff
@@ -2,10 +2,25 @@
 // maar ook een manier moeten vinden om hetgeen dat je verzamelt ergens te bundelen. Op deze manier zul je ontdekken hoe je omgaat met scope. Pak vooral het hoofdstuk op EdHub over for-loops er nog eens bij!
 // Tip: je mag hier geen ingebouwde object methoden gebruiken, dus daar hoef je niet naar te kijken.
 
-const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
-
+const klas1 = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+const klas2 = [6, 4, 5];
+const klas3 =  [8, 9, 4, 6, 10];
+let cumLaudeGrades
 /* Opdracht  1: Cum Laude */
-
+console.log(getCumlaudGrades(klas1));
+console.log(getCumlaudGrades(klas2));
+console.log(getCumlaudGrades(klas3));
+
+function getCumlaudGrades(arr) {
+    let counter = 0;
+    for (let i = 0; i < arr.length; i++) {
+        if(arr[i] > 7 ){
+            counter += 1;
```

Comment:
Je zou dit nog wat kunnen versimpelen
```suggestion
            counter++;
```